SecondYearFirstSemestr/Test3/Test3/Reflector.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.Reflection;
+using System.Text;
+
+namespace Test3
+{
+	public static class Reflector
+	{
+
+		//Считывает данные класса и записывает их в файл
+		public static void PrintStructure(Type someClass)
+		{
+			var data = getStructureInString(someClass);
+			Console.WriteLine(data);
+			File.WriteAllText(someClass.Name + ".cs", data);
+		}
+
+		public static void DiffClassese(Type a, Type b)
+		{
+			throw new NotImplementedException();
+		}
+
+
+		// Считывает данные класса и превращает в строку
+		private static string getStructureInString(Type someClass)
+		{
+			if (!someClass.IsClass && !someClass.IsInterface)
+			{
+				throw new ArgumentException();
+			}
+
+			var builder = new StringBuilder();
+			if (someClass.IsClass)
+			{
+				builder.Append("public class " + someClass.Name);
+				if (someClass.ContainsGenericParameters)
+				{
+					var args = someClass.GetGenericArguments();
+					foreach(var arg in args)
+					{
+						builder.Append(arg.Name+ "");
```

Comment:
```suggestion
						builder.Append(arg.Name + "");
```
Как Вы так брутально нарушаете стиль кодирования? Ни одна среда разработки так не позволит :)